server.go

```diff
@@ -69,20 +87,24 @@ func main() {
 
 	// Setup static files
 	static := web.New()
-	publicPath := application.Config.Get("general.public_path").(string)
 	static.Get("/assets/*", http.StripPrefix("/assets/",
-		http.FileServer(http.Dir(publicPath))))
```

Comment:
Could you check out my Nov 23 commit to the "stuff" branch?
https://github.com/chappjc/dcrstakepool/blob/0c54539aa59ff27cc17f0fcf05e92a45f1abc202/server.go#L60-L69

I managed to break the static files route when I moved off the default http router, and I realized that using `http.Handle` (two lines down) and using `server := &http.Server{Handler: app}` + `server.Serve()` at the end were incompatible.  Does this work for you?  I found it necessary to create a static files route `within` the `app` router.

```go
	// Setup static files
	publicPath := application.Config.Get("general.public_path").(string)
	assetHandler := http.StripPrefix("/assets/",
		http.FileServer(http.Dir(publicPath)))
	// static := web.New()
	// static.Get("/assets/*", assetHandler)

	// Apply middleware
	app := web.New()
	app.Handle("/assets/*", assetHandler)
```